Examples/MAX78000/CNN/facial_recognition/README.md

```diff
@@ -0,0 +1,208 @@
+# MAX78000 Facial Recognition Demo
+
+
+
+## Overview
+This demo includes **FaceDetection** and **FaceID** CNN models and runs them sequentially on MAX78000 Feather board only. The **FaceID** CNN model weight binary file (Include/weigths_2.bin) needs to be programmed to SD card.
+The **FaceDetection** CNN model detects a face and application draws a box around the face.
+
+The **FaceID** CNN model demonstrates identification of a number of persons from their facial images.
+
+For this purpose, the **FaceID** CNN model generates a 512-length embedding for a given image, whose distance to whole embeddings stored for each subject is calculated. The image is identified as either one of these subjects or 'Unknown' depending on the embedding distances.
+
+## Facial Recognition Demo Software
+
+### Building firmware
+
+Navigate directory where Facial Recognition demo software is located and build the project:
+
+```bash
+$ cd /Examples/MAX78000/CNN/facial_recognition
+$ make
+```
+
+If this is the first time after installing tools, or peripheral files have been updated, first clean drivers before rebuilding the project: 
+
+```bash
+$ make distclean
+```
+
+**Note: If you are using Eclipse, please also make sure to change the value of Board environment variable to "FTHR_RevA by:**
+
+*right click project name > Properties > C/C++ Build > Environment > Board"*
+
+<img src="Resources/eclipse_board.png" style="zoom:33%;" />
+
+
+
+### Load firmware image to MAX78000 Feather
+
+Connect USB cable to CN1 USB connector.
+
+If you are using Windows, load the firmware image with OpenOCD in a MinGW shell:
+
+```bash
+openocd -s $MAXIM_PATH/Tools/OpenOCD/scripts -f interface/cmsis-dap.cfg -f target/max78000.cfg -c "program build/MAX78000.elf reset exit"
+```
+
+If using Linux, perform this step:
+
+```bash
+./openocd -f tcl/interface/cmsis-dap.cfg -f tcl/target/max78000.cfg -c "program build/MAX78000.elf verify reset exit"
+```
+
+
+
+### Write FaceID model weights to SD Card
```

Comment:
I think this section should come first in Building firmware sections:  something like:
Building Firmware:
1) Building FaceID weight writer firmware: 
Use **facial_recognition/SDHC_weights** project and it will create **weights_2.bin** file in binary format. Make sure to have a formatted SD card inserted into the Feather board.
$ cd /Examples/MAX78000/CNN/facial_recognition/SDHC_weights
$ make

Next, load this firmware image to MAX78000 Feather as described in the following section to write the binary FaceID weights to the SD card.

2) Building Facial recognition firmware:
Navigate the directory where Facial Recognition demo software is located and build the project: ....

Reply:
Updated